extensions/amp-vi-player/validator-amp-vi-player.protoascii

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-vi-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-vi-player"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-vi-player>
+  html_format: AMP
+  tag_name: "AMP-VI-PLAYER"
+  requires_extension: "amp-vi-player"
+  attr_lists: "extended-amp-global"
+  attrs: {
+    name: "data-publisher-id"
+    mandatory: true
+  },
+  attrs: {
+    name: "data-channel-id"
```

Comment:
We prefer attrs in alphabetical order - please move data-channel-id before data-publisher-id.

Reply:
Obsolete.